kubernetes-client/src/main/resources/META-INF/services/io.fabric8.kubernetes.client.extension.ExtensionAdapter

```diff
@@ -13,46 +13,3 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
-
```

Comment:
Does it make sense to keep this file and SPI?

Reply:
I'm fine with removing it.

Reply:
```
Error:  Bundle io.fabric8:kubernetes-client:jar:6.0-SNAPSHOT : Input file does not exist: target/classes/META-INF/services/io.fabric8.kubernetes.client.extension.ExtensionAdapter
Error:  Error(s) found in bundle configuration
```
Reference needs to be remved too

Reply:
Updated now.  I look forward to the day when osgi support is dropped, or at least pushed out as a concern of the core project.

Reply:
:pensive: